cypress/integration/manifest.js

```diff
@@ -1,9 +1,16 @@
 import { testResponseCode } from '../support/metaTestHelper';
+import describeForLocalOnly from '../support/describeForLocalOnly';
 
-describe('Manifest.json files', () => {
-  ['news', 'persian', 'igbo', 'pidgin', 'yoruba'].forEach(service => {
-    it(`should return a 200 status code for ${service}`, () => {
-      testResponseCode(`/${service}/articles/manifest.json`, 200);
-    });
+const testManifest200s = service => {
+  it(`should return a 200 status code for ${service}`, () => {
+    testResponseCode(`/${service}/articles/manifest.json`, 200);
   });
+};
+
+describe('Manifest.json files', () => {
+  ['news', 'persian'].forEach(testManifest200s);
```

Comment:
Does this pass the service as a parameter to the arrow function for each iteration?